apps/examples/testcase/le_tc/kernel/tc_mqueue.c

```diff
@@ -596,6 +595,9 @@ static void tc_mqueue_mq_open_close_send_receive(void)
 	status = pthread_attr_setschedparam(&attr, &sparam);
 	TC_ASSERT_EQ("pthread_attr_setschedparam", status, OK);
 
+#ifdef CONFIG_SMP
+	CPU_SET(0, &(attr.affinity));
+#endif
```

Comment:
Actually this is an app. Do we need to call this here?

Reply:
Previously we were facing issues in this testcase when smp is enabled. But, after the final smp changes in kernel, the testcase is passing without this change. So i will revert it.